webapp/src/modules/reportList/report-list.query.js

```diff
@@ -0,0 +1,11 @@
+import gql from 'graphql-tag';
+
+export const REPORTS = gql`
```

Comment:
This name is to ass.

Reply:
Przepraszam za tego suchara #angielskizTuskiem. Chodziło o to, że jest to rzici.